src/pages/index.mdx

```diff
@@ -12,35 +8,134 @@ The developer docs will teach you how to build applications, issue and use token
+| **Learn** | [Operations & transactions](https://developers.stellar.org/docs/learn/fundamentals/transactions) | Wallets build and submit transactions, learn about about how operations and transactions work here |
+| **Learn** | [Signatures & multisig](https://developers.stellar.org/docs/learn/encyclopedia/security/signatures-multisig) | Signatures are authorization for transactions on the Stellar network, learn about them here |
+| **Learn** | [Anchor overview](https://developers.stellar.org/docs/learn/fundamentals/anchors) | Wallets may want to connect to financial rails via anchors (on- and off-ramps), learn about them here |
+| **Tutorial** | [Wallet SDK](https://developers.stellar.org/docs/category/build-a-wallet-with-the-wallet-sdk) | Learn to build a wallet with the Wallet SDK in Kotlin, Flutter, or TypeScript |
+| **Data** | [Horizon introduction](https://developers.stellar.org/docs/data/horizon) | Access network data using the Horizon REST API |
+
+| **Decentralized applications** | _Start by viewing the smart contract wayfinding section_ |  |
+| :-: | --- | --- |
+| **Learn** | [Transaction simulation](https://developers.stellar.org/docs/learn/encyclopedia/contract-development/contract-interactions/transaction-simulation) | A process that allows developers to test the outcome of a transaction without actually executing it, helping to ensure accuracy and prevent errors |
+| **Guide** | [Dapp frontend](https://developers.stellar.org/docs/build/apps/dapp-frontend) | Create a web app that interacts with the contracts via RPC calls |
+| **Guides** | [Dapp how-to guides](https://developers.stellar.org/docs/build/guides/dapps) | Explore the various guides relating to dapp development |
+| **Tool** | [Stellar Wallets Kit](https://stellarwalletskit.dev/) | A kit to handle all Stellar Wallets at once with a simple API and without caring about individual configurations for each one of them |
+| **Data** | [Stellar JS SDK](https://stellar.github.io/js-stellar-sdk/) | Communicate with a Horizon server and Soroban RPC |
+| **Data** | [RPC introduction](https://developers.stellar.org/docs/data/rpc) | Access network data using the RPC |
+
+| **Exchanges** |  |  |
+| :-: | --- | --- |
+| **Tools** | [SDKs](https://developers.stellar.org/docs/tools/sdks) | Used to connect to Horizon or RPC, as well as construct and parse transactions |
+| **Data** | [Run your own RPC](https://developers.stellar.org/docs/data/rpc/admin-guide) | Exchanges typically run their own RPC or Horizon to maintain greater control over data integrity, performance, and security |
+| **Data** | [Run your own Horizon](https://developers.stellar.org/docs/data/horizon/admin-guide) | Exchanges typically run their own RPC or Horizon to maintain greater control over data integrity, performance, and security |
+
+### Stellar for infrastructure providers
```

Comment:
Hmm should Hubble also be included in the `infra provider` persona?

Or is there any interest in an `analytics` persona which would list Hubble/external analytics platforms like Dune?

Reply:
I think the Hubble sections could be like

* Learn
 * Connecting to Hubble | https://github.com/stellar/stellar-docs/blob/main/docs/data/hubble/analyst-guide/connecting.mdx
 * Data Catalog | https://github.com/stellar/stellar-docs/tree/main/docs/data/hubble/data-catalog
* Tutorial
 * Admin Guide | https://github.com/stellar/stellar-docs/tree/main/docs/data/hubble/admin-guide
 * Analyst Guide | https://github.com/stellar/stellar-docs/tree/main/docs/data/hubble/analyst-guide
 * End to End Example | https://github.com/stellar/stellar-docs/blob/main/docs/data/hubble/analyst-guide/end-to-end-analysis-example.mdx
* Tools
 * Analytics Platforms | https://github.com/stellar/stellar-docs/blob/main/docs/data/hubble/analytics-platforms.mdx

Reply:
What would the difference between Analytics and Infra Providers be? Right now Infra Providers is "I want to set up a __ (Horizon, RPC) for others to leverage for their use case." What would Analytics provide?

Reply:
Hmm it is a little mixed between `Analytics and Infra Providers`. Hubble could fall in both in the sense that an Infra Provider can run their own Hubble/Hubble-like instance (e.g., Hubble Admin Guide). And I would say for `Analytics` it'd be more like "I just want to be able to write SQL to do complex analysis on network data". I'm not sure if I were a Business Analyst I'd think to look for `Analytics/Hubble` within the `Infra Providers` section

My overall goal is just to include `Hubble` in wayfinding one way or another

Reply:
Makes sense. I think I'm leaning toward creating an Analytics persona section.